package.json

```diff
@@ -1944,10 +1959,15 @@
           "group": "4_flink@1"
         },
         {
-          "command": "confluent.uploadArtifact",
-          "when": "view in confluent.viewsWithResources && (viewItem =~ /.*-kafka-cluster.*/ || viewItem =~ /.*compute-pool*/) && config.confluent.flink.artifacts",
```

Comment:
There wasn't any reason for this to span multiple views (`view in confluent.viewsWithResources`) since we only ever show Kafka clusters and Flink compute pools in the Resources view.